demo/px-data-grid-height-demo.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
```

Comment:
Could you please explain, why this demo does not follow the style of px-components?
Example:
https://github.com/vaadin/px-data-grid/blob/master/demo/px-data-grid-demo.html

Reply:
Oh, haven't noticed the second part of the first sentence of @limonte's comment, sorry.